src/components/faqs.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+
+const faqs = [
```

Comment:
Crear interfaz FAQ, esta tiene que tener:
- id: number/string (decidi vos)
- question: string
- answer: string

```suggestion
const faqs: FAQ[] = ...
```